update RSKj node version (#990)
rsksmart/devportal: content/rsk-devportal/libraries/rsk-precompiled-abis/index.md

tags: [libraries, abi, pre-compiled]
---

> Welcome to the precompiled ABIs from RSK!

Reviewer: ```suggestion
Welcome to the precompiled ABIs from Rootstock!
```

Here you will find the ABIs for the existing precompiled contracts in RSK. You will also get their addresses and a builder to use it with web3js.

## Version

> Different versions of the package mentioned are required for different RSK releases.

Reviewer: ```suggestion
Different versions of the package mentioned are required for different releases.
```

> The semantic versioning of this package doesn’t correlate to the semantic versioning of RSK. For each named release of RSK, there will be a corresponding name version in npm.

Reviewer: ```suggestion
The semantic versioning of this package doesn’t correlate to the semantic versioning of Rootstock. For each named release of the node, there will be a corresponding name version in npm.
```

| 2.0.0-WASABI    | WASABI-1.0.0  |
| 2.0.1-WASABI    | WASABI-1.0.0  |
| 3.0.0-PAPYRUS   | PAPYRUS-2.0.0 |

| Package Version | Rootstock version   |
|-----------------|---------------|
| 1.0.0-ORCHID    | ORCHID-0.6.2  |
| 2.0.0-WASABI    | WASABI-1.0.0  |
| 2.0.1-WASABI    | WASABI-1.0.0  |
| 3.0.0-PAPYRUS   | PAPYRUS-2.0.0 |
| 4.0.0-IRIS      | IRIS-3.0.0    |
| 4.0.2-IRIS      | IRIS-3.0.0    |
| 5.0.0-HOP       | HOP-4.0.0    |
| 5.0.0-FINGERROOT | FINGERROOT-5.0.0 |

> | 6.0.0-ARROWHEAD | ARROWHEAD-6.0.0 |

Author: I Redirected https://dev.rootstock.io/libraries/rsk-precompiled-abis/ to https://dev.rootstock.io/develop/libs/rsk-precompiled-abis/ in the node version update PR. This eliminates the need to maintain both separately and resolved all issues

> | 6.0.0-ARROWHEAD | ARROWHEAD-6.0.0 |

Reviewer 2: Perfect, thanks.
